Dockerfile

```diff
@@ -1,5 +1,19 @@
 FROM node:current-alpine
 
+ARG GIT_COMMIT
+ARG WORKBENCH_CLIENT_VERSION="latest"
+
+LABEL maintainer="Charles Alleman <alleman@qut.edu.au>" \
+      description="Production environment for workbench client server" \
+      version="3.0" \
```

Comment:
how is `version` different from `org.ecosounds.version`

Reply:
Was just following the template here: https://github.com/QutEcoacoustics/audio-analysis/blob/master/build/Dockerfile#L7-L16. No clue what `org.ecosounds.version` is, and what its relevance is to

Reply:
in which case, if they're the same, set them both the version build arg

Reply:
The matching labels are `version` and `org.ecosounds.schema-version`. `org.ecosounds.version` is slightly different. That said, unless i know what the labels are for, it seems like almost all of them are useless and we should instead only set the following:

- `maintainer`
- `description`
- `version`
- `name`
- `vendor`
- `url`
- `ref`